contrib/windows-cmake/CMakeLists.txt

```diff
@@ -3,48 +3,105 @@ project(hwloc
     LANGUAGES C
     VERSION 2.7.0)
 
-# Available configuration options
-# HWLOC_SKIP_LSTOPO doesn't build/install lstopo
-# HWLOC_SKIP_TOOLS doesn't build/install other hwloc tools
-# HWLOC_SKIP_INCLUDES doesn't install headers
+enable_testing()
 
-set(TOPDIR ../..)
+option(HWLOC_ENABLE_TESTING "Enable testing" ON)
+option(HWLOC_SKIP_LSTOPO "don't build/install lstopo")
+option(HWLOC_SKIP_TOOLS "don't build/install other hwloc tools")
+option(HWLOC_SKIP_INCLUDES "don't install headers")
 
-# Configure based on the MSVC config
+set(TOPDIR ${PROJECT_SOURCE_DIR}/../..)
+
+set(CMAKE_STATIC_LIBRARY_PREFIX lib)
+set(CMAKE_SHARED_LIBRARY_PREFIX lib)
```

Comment:
```suggestion
```
simply no.  Windows doesn't need lib prefixes and the autotool build also does not have it on windows.

Reply:
OK then I will remove them. The prior naming windows-cmake scheme was outputting library files "liblibhwloc.*" on non-MSVC.